manuscript/registry/dockerhub.md

```diff
@@ -1,4 +1,4 @@
-# Store Container Image in a Registry with Distribution
+# Store Container Image in a Registry with Docker Hub
```

Comment:
I thought to keep it as "Distribution" since that is the CNCF project. It's used as a base on top of which people build container registries. Since it's not useful alone (it's a building block), the commands themselves are based on Docker Hub since that is one of the registries built on top of Docker Hub.

Reply:
I'll work it into the registry intro.  But it still makes sense to the Docker Hub solution 'DockerHub' in the filename imho since both of our registries are Distribution-based

Reply:
Would that mean that we talk about Distribution in both of the choices?

Reply:
I think so, or in the introduction part of the registry page.  I'll suss it out.